bedrock/firefox/templates/firefox/whatsnew/index.html

```diff
@@ -45,6 +45,11 @@ <h1 class="wnp-main-title">{{ ftl('whatsnew-s2d-download-firefox-for-your') }}</
     </ul>
   </div>
 </section>
+
+<div class="wnp-footer">
+  <p class="wnp-sign-off">{{ ftl('whatsnew-signoff') }}</p>
+  {% include "firefox/whatsnew/includes/mofo-donate-cta.html" %}
```

Comment:
It just occurred to me: if this is being added to the fallback page shown to all locales, it should only appear if the string is translated. Not sure how best to do that with some locales hard-coded... maybe something like:
```suggestion
{% if LANG in ["en-US", "en-CA", "en-GB", "fr", "de", "es-ES", "it", "pl"] or ftl_has_messages('whatsnew-donate') %}
  {% include "firefox/whatsnew/includes/mofo-donate-cta.html" %}
{% endif %}
```

Reply:
Nice catch!